src/HomePage.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * @file HomePage.js
+ *
+ * @brief Home page
+ *
+ * @author Gabriel Kishida <gabriel.kishida@thunderatz.org>
+ * @author Vanderson Santos <vanderson.santos@thunderatz.org>
+ *
+ * @date 06/2021
+ */
+
+import React,{useEffect} from 'react';
+import {
+  SafeAreaView,
+  
+} from 'react-native';
+
+import { Navigation } from "react-native-navigation";
+
+import {H1,H2,H3,Body,BodySecondary} from './components/typography'
+import {Header, CellContainer, CellTitleContainer, VBox, VSeparator, TextInputCell} from './components/cell'
+import {PrimaryButton, SecondaryButton} from './components/button'
+import {SmallModal} from './components/modal'
+import {BTPostData} from './bt_communication/bt_data_sender'
+import {BTGetDataContinuous,BTGetData} from "./bt_communication/bt_data_receiver"
```

Comment:
Tem algum motivo especial pra esse ser em aspas duplas?  Acho q pode deixar igual aos outro n?
```suggestion
import {BTGetDataContinuous,BTGetData} from './bt_communication/bt_data_receiver'
```